lib/pyld/documentloader/requests.py

```diff
@@ -55,12 +58,16 @@ def loader(url, options={}):
                     'the URL\'s scheme is not "https".',
                     'jsonld.InvalidUrl', {'url': url},
                     code='loading document failed')
-            headers = options.get('headers')
-            if headers is None:
-                headers = {
-                    'Accept': 'application/ld+json, application/json'
-                }
-            response = requests.get(url, headers=headers, **kwargs)
+            
+            headers = {} if 'headers' in kwargs else ({
+                'headers': (
+                    options.get('headers', None)
```

Comment:
Why we're addressing both `kwargs['headers']` and `options['headers']`, where should those be found?